assets/js/modules/reader-revenue-manager/components/dashboard/RRMIntroductoryOverlayNotification.js

```diff
@@ -0,0 +1,191 @@
+import { createInterpolateElement } from '@wordpress/element';
+import SupportLink from '../../../../components/SupportLink';
+
+const { ONBOARDING_COMPLETE } = PUBLICATION_ONBOARDING_STATES;
+
+export const RRM_MONETIZATION_OVERLAY_NOTIFICATION =
+	'rrmMonetizationOverlayNotification';
+
+function RRMIntroductoryOverlayNotification() {
+	const viewContext = useViewContext();
+	const isViewOnly = useViewOnly();
+	const dashboardType = useDashboardType();
+
+	const { publicationID, publicationOnboardingState, paymentOption } =
+		useSelect(
+			( select ) =>
+				select( MODULES_READER_REVENUE_MANAGER ).getSettings() || {}
+		);
+
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed(
+			RRM_MONETIZATION_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const { dismissOverlayNotification } = useDispatch( CORE_UI );
+
+	const usingNonMonetaryCTA =
+		paymentOption === '' || paymentOption === 'noPayment';
+
+	const shouldShowNotification =
+		isDismissed === false &&
+		! isViewOnly &&
+		dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD &&
+		publicationOnboardingState === ONBOARDING_COMPLETE &&
+		usingNonMonetaryCTA;
+
+	const isDismissing = useSelect( ( select ) =>
+		select( CORE_USER ).isDismissingItem(
+			RRM_MONETIZATION_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const serviceURL = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getServiceURL( {
+			path: 'reader-revenue-manager',
+			query: {
+				publication: publicationID,
+			},
+		} )
+	);
+
+	const dismissNotice = () => {
+		dismissOverlayNotification( RRM_MONETIZATION_OVERLAY_NOTIFICATION );
+	};
+
+	const dismissNotification = () => {
+		trackEvent(
+			`${ viewContext }_rrm-monetization-notification`,
+			'dismiss_notification'
+		).finally( () => {
+			dismissNotice();
+		} );
+	};
+
+	return (
+		<OverlayNotification
+			className="googlesitekit-reader-revenue-manager-overlay-notification googlesitekit-reader-revenue-manager-monetization-notification"
+			GraphicDesktop={ ReaderRevenueManagerIntroductoryGraphicDesktop }
+			GraphicMobile={ ReaderRevenueManagerIntroductoryGraphicMobile }
+			onShow={ () => {
+				trackEvent(
+					`${ viewContext }_rrm-monetization-notification`,
+					'view_notification'
+				);
+			} }
+			shouldShowNotification={ shouldShowNotification }
+			notificationID={ RRM_MONETIZATION_OVERLAY_NOTIFICATION }
+		>
+			<div className="googlesitekit-overlay-notification__body">
+				<h3>
+					{ paymentOption === 'noPayment'
+						? __(
+								'New! Monetize your content with Reader Revenue Manager',
+								'google-site-kit'
+						  )
+						: __(
+								'Complete account setup with Reader Revenue Manager',
+								'google-site-kit'
+						  ) }
+				</h3>
+				<p>
+					{ paymentOption === 'noPayment'
+						? createInterpolateElement(
+								__(
+									'Now, you can offer your users subscription options to access content behind a paywall, or make voluntary contributions. <a>Learn more.</a>',
```

Comment:
Let's stick with the ACs regarding using the comma (`,`) after "_Now_". I did notice the comment in Figma and it looks like we'll not be updating it. Also, the "_Learn more_" link should not have a period or external indicator afterwards.

```suggestion
									'Now you can offer your users subscription options to access content behind a paywall, or make voluntary contributions. <a>Learn more</a>',
```